app/controllers/devise_token_auth/confirmations_controller.rb

```diff
@@ -6,21 +6,23 @@ def show
       @resource = resource_class.confirm_by_token(params[:confirmation_token])
 
       if @resource.persisted?
-        expiry = nil
+        lifespan = nil
+        cost     = nil
         if defined?(@resource.sign_in_count) && @resource.sign_in_count > 0
-          expiry = (Time.zone.now + 1.second).to_i
+          lifespan = 1 # 1 second
+          cost     = 4 # 4 is the minimum cost
```

Comment:
this part might be better (with the token creation) in another method because this action is getting big

Reply:
I think we should hold off refactoring in favour of #1075 or something better.

Reply:
Sorry, I thought that PR was dead, ignore this comment